ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,37 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
+ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+ RUN add-apt-repository "deb https://repo.sovrin.org/ursa/deb xenial master"
```

Comment:
You need to add repo by:
```
echo "deb https://repo.sovrin.org/ursa/deb xenial master" >> /etc/apt/sources.list && \
    apt-get update
```
or install software-properties-common package

Reply:
So, Ursa is in a different sovrin repo channel. How they installation of indy-node and plenum (from deb) should look like now? Does it mean that everyone will have to add a new repo?
This looks like a breaking change affecting all installations and upgrade procedures.